src/components/dataset/overview/dataset-properties.vue

```diff
@@ -95,7 +95,7 @@ export default {
 <i18n lang="json5">
   {
     "en": {
-      "emptyTable": "There are no Properties in this Dataset."
+      "emptyTable": "The Entities in this Entity List do not have any user-defined Properties."
```

Comment:
Should we capitalize "properties"? I had been thinking that we wouldn't, because we generally don't capitalize "fields", which is a similar concept for forms. I guess we were capitalizing here already. But I see other places where we aren't capitalizing:

- `propertiesCount` in en.json5
- `FormDraftPublish` component ("a new property")
- `FormDraftTesting` ("the desired properties")

No preference here around whether we capitalize or not. Just wanted to note that we don't for fields and that there are a few places where we don't for properties.

Reply:
Let's not! I went back and forth on this one.